database-ms/src/main/java/org/dice_research/sask/database_ms/DbMSApplication.java

```diff
@@ -0,0 +1,16 @@
+package org.dice_research.sask.database_ms;

```

Comment:
This class does not seem to be formatted according to the rules

Reply:
The structure of the this class is similar to the other micro services application class.